docs/development_suite/identity-and-access-management/service-account-management.md

```diff
@@ -0,0 +1,126 @@
+---
+id: service-account-management
+title: Manage Service Accounts
+sidebar_label: Manage Service Accounts
+---
+
+Mia-Platform Console allows the creation of Service Accounts, which are typically used for automated processes or to allow a service to access resources on behalf of multiple Users.  
+Note that new Service Accounts can only be created at Company level. For each Project or Environment they will be visible in the Identities section.
+Just like human Users, Service Accounts can be assigned Roles on Company, Project or Runtime Environment level, based on which they will be able to perform different types of action.  
+
+:::note
+To find out more about Roles check out the available [Capabilities](/development_suite/identity-and-access-management/console-levels-and-permission-management.md#users-capabilities-inside-console) that can be assigned to an identity.
+:::
+
+### What can you do with a Service Account?
+
+Virtually anything a User can do inside the Console can be entrusted to a dedicated Service Account, which will perform the desired actions automatically.
+
+For instance, a Service Account could be granted permissions to make automatic deploys, which can greatly streamline the process of deploying code changes to an application. The Service Account could use its credentials to access the appropriate deployment resources and automatically deploy new versions of the application as soon as new code is pushed to the source code repository.  
```

Comment:
```suggestion
For instance, you can grant a Service Account the permissions to make automatic deploys: this can significantly streamline the whole process of deploying code changes to an application. The Service Account could use its credentials to access the appropriate deployment resources and automatically deploy new versions of the application as soon as new code is pushed to the source code repository.  
```